setup.py

```diff
@@ -40,15 +40,26 @@ def read(fname):
     
     install_requires=[
         'pytest >=3.1.1',
+        ########## lint stuff
+        'pyflakes',
```

Comment:
Is this really needed - aren't we vendoring it in at this point?

Reply:
Yes, it's needed - there's more to pyflakes than this file...like all the rules etc! :) 

We only modify one pyflakes file, to return more info from flake checking and to support `noqa` (which pyflakes doesn't support - we could use another linter that does, but I don't think we even run another linter on any of our code, let alone notebooks...).